config/template/sequences_leveling/default_sequence.json

```
@@ -0,0 +1,231 @@
+{
+	"normal": {
+		//Quests runs
+		"quests" : [
+			/* Run parameters
+			{
+				"run": "name", //Name associated with run in run.go BuildRun function
+				"minLevel": 10, //Minimum level to start this run
+				"maxLevel": 20, //Maximum level to start this run
+				"lowGoldRun": true, //If true, start run only if gold < lvl*1000
+				"skipTownChores": true, //If true, don't do town stuff after this run
+				"exitGame": true, //If true, exit current game at the end of this run
+				"stopIfCheckFails": true, //If true, don't try executing following runs if requirements check fails
+				"parameters": "{}" //Provide additional parameters to the run (none implemented for now)
+			}
+			*/
+			
+			{"run":"den"},
+			{"run":"bloodraven"},
+			{"run":"rescue_cain"},
+			{"run":"countess"},
+			{"run":"retrieve_hammer"},
+			{"run":"andariel"},
+			
+			{"run":"radament", "minLevel":15},
+			{"run":"cube"},
+			{"run":"staff"},
+			{"run":"amulet"},
+			{"run":"summoner"},
+			{"run":"duriel", "minLevel":24, "StopIfCheckFails": true},
```

```suggestion
			{"run":"duriel", "minLevel":24, "stopIfCheckFails": true},
```

it should be in camel case